Makefile

```diff
@@ -512,6 +511,7 @@ prd/embed.html: src/index.mako.html \
 prd/img/: src/img/*
 	mkdir -p $@
 	cp -R $^ $@
+	cp -f $@/favicon.ico $@/../
```

Comment:
All img folder is copied into prd/   so I just need to copy the favicon from prd/img to prd/